source/js/components/currency-selector.js

```diff
@@ -6,6 +6,9 @@ class CurrencySelect {
   constructor(node) {
     this.selectMenu = node;
     this.formContainer = document.getElementById("js-donate-form");
+    this.locale = this.formContainer
+      .getAttribute("data-locale")
+      .replace("_", "-");
```

Comment:
@solarissmoke I would suggest adding a note here to clarify why this transformation is necessary